playwright.config.ts

```diff
@@ -0,0 +1,73 @@
+import { defineConfig, devices } from "@playwright/test";
+import { config } from "dotenv";
+
+config();
```

Comment:
are we using `.env` or `.env.local` for non-public env vars? if the latter, this needs to be adjusted